src/AWS.Deploy.CLI/Commands/DeployCommand.cs

```diff
@@ -85,6 +89,23 @@ public async Task ExecuteAsync(string stackName, UserDeploymentSettings? userDep
             await CreateDeploymentBundle(orchestrator, selectedRecommendation, cloudApplication);
 
             await orchestrator.DeployRecommendation(cloudApplication, selectedRecommendation);
+
+            var displayedResources = await _displayedResourcesHandler.GetDeploymentOutputs(cloudApplication, selectedRecommendation);
+            DisplayOutputResources(displayedResources);
+        }
+
+        private void DisplayOutputResources(List<DisplayedResourceItem> displayedResourceItems)
+        {
+            _toolInteractiveService.WriteLine("Created AWS Resources");

```

Comment:
"Created AWS Resources" -> "Resources"
The reason being if we are doing an redeployment then we didn't just created the resources. It is also possible via saved CDK projects and CloudFormation custom resources to create non aws resources.

Reply:
Fixed